rpc/documentation/api.md

```diff
@@ -1559,6 +1560,42 @@ signature algorithms other than secp256k1.
 
 ___
 
+#### `ValidateAddress`
+
+The `ValidateAddress` method verifies if an address is valid.
+
+**Request:** `ValidateAddressRequest`
+
+- `string address`: The address to be validated.
+
+**Response:** `ValidateAddressResponse`
+
+- `bool is_valid`: True if valid, false if not.
+
+- `string address`: The validated address.
+
+- `bool is_mine`: True if the address is an address of querying wallet, false if not.
+
+- `bool is_watch_only`: True if the querying wallet is watch-only, false if not.
+
+- `string pub_key_addr`: The public key address.
+
+- `string pub_key`: The hex encoded string public key associated with the address.
+
+- `string is_compressed`: True if address uses a compressed format, false if not.
+
+- `string account`:  The account name of the wallet owning the address.
+
+- `repeated string addresses`: A collection of addresses extracted from the address if it's a script.
+
+- `string hex`: The hex encoded redeem script if the address is p2sh
+
+- `string script`: The script type
+
+- `string sigs_required`: The number of signatures required for the address.
```

Comment:
don't use strings as numbers :)

this is just a doco issue, the actual message is using int32